WooCommerce/Classes/ViewRelated/ListSelector/PaginatedListSelectorViewController.swift

```diff
@@ -235,10 +235,26 @@ private extension PaginatedListSelectorViewController {
         tableView.rowHeight = UITableView.automaticDimension
         tableView.backgroundColor = .listBackground
 
+        tableView.separatorStyle = viewProperties.separatorStyle
+
+        if viewProperties.tableViewStyle == .plain {
+            // Removes extra header spacing in ghost content view.
+            tableView.estimatedSectionHeaderHeight = 0
+            tableView.sectionHeaderHeight = 0
+        }
```

Comment:
Seems that this condition need to be executed always, also when the table view style is grouped, or this is what happens if compared to develop:

| Develop - ghost tableview            |  This branch - ghost tableview |
:-------------------------:|:-------------------------:
![Simulator Screen Shot - iPhone 11 Pro - 2020-07-14 at 14 33 40](https://user-images.githubusercontent.com/495617/87427292-dda2c680-c5e0-11ea-8321-6e4ff0ffb83a.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-07-14 at 14 27 31](https://user-images.githubusercontent.com/495617/87427282-da0f3f80-c5e0-11ea-9de2-d60d74661c1e.png)

This happens both in Tax classes and Shipping classes.

Reply:
Thanks @pmusolino for noticing this issue! yes it does look like this is needed for grouped table view style as well. updated in https://github.com/woocommerce/woocommerce-ios/pull/2492/commits/a9485bdf4a222c40671b2eadf3e720662b23e071 and screenshots on price tax classes:

`develop` | PR branch
-- | --
![Simulator Screen Shot - iPhone 11 - 2020-07-15 at 09 47 08](https://user-images.githubusercontent.com/1945542/87494217-08445c00-c681-11ea-80e7-ba4548357386.png) | ![Simulator Screen Shot - iPhone 11 - 2020-07-15 at 09 49 51](https://user-images.githubusercontent.com/1945542/87494220-0aa6b600-c681-11ea-92c6-72a5de169f64.png)